consumer/videooutput.cpp

```diff
@@ -32,6 +32,9 @@ VideoOutput::~VideoOutput() {
 }
 
 void VideoOutput::consumerFormat( ProducerFormat format ) {
+
+    qDebug() << "CONSUMER FORMAT START";

```

Comment:
Didn't you make a big stink a long time ago about making sure to tag debug messages? :)